src/ad-networks.ts

```diff
@@ -20,6 +17,7 @@ import { Application, Challenge, User, Withdrawal } from './types/schema'
 
 // Respond to application white listed events
 export function applicationWhitelist(event: _ApplicationWhitelisted): void {
+  let foo = new Bytes()
```

Comment:
What's this doing?

Reply:
Haha, erm, I'll remove this.

Reply:
I think that was from testing removing the empty constructors. 😉